lib/sequencescape-api/rails.rb

```diff
@@ -26,7 +25,7 @@ def api_class
     private :api_class
 
     def configure_api
-      @api = api_class.new({ cookie: cookies['WTSISignOn'] }.merge(api_connection_options))
+      @api = api_class.new(api_connection_options)
```

Comment:
`api_connection_options` is just an empty hash, defined below in `def api_connection_options` ?

Reply:
Its the responsibility of the application including sequencescape_client_api to implement this, although it would probably make more sense to throw an exception explaining this. Will change.

Reply:
Thanks